classes/task/FileLoader.inc.php

```diff
@@ -126,7 +127,39 @@ function executeActions() {
 		if (!$this->checkFolderStructure()) return false;
 
 		$foundErrors = false;
+		// get setting value
+		$memoryLimit = ini_get('memory_limit');
+		// multiply for shorthand notation
+		switch (strtolower($memoryLimit[strlen($memoryLimit)-1])) {
```

Comment:
Doesn't this try to perform math on the number without first stripping off the suffix?